test/extended/apiserver/patch.go

```diff
@@ -115,7 +114,23 @@ func createWellKnownKubeAPIServerOperatorResource(ctx context.Context, resourceC
 	kasOperator := unstructuredToKubeAPIServerOperator(unstructuredKasOperator.Object)
 	kasOperatorFromManifest := unstructuredToKubeAPIServerOperator(unstructuredKasOperatorManifest.Object)
 	kasOperator.Status = kasOperatorFromManifest.Status
-	unstructuredKasOperator, err = resourceClient.UpdateStatus(ctx, kubeAPIServerOperatorToUnstructured(kasOperator), metav1.UpdateOptions{})
+	obj := kubeAPIServerOperatorToUnstructured(kasOperator)
+	// We are not going to allow setting NodeStatus.CurrentRevision on creation of a NodeStatus.
```

Comment:
The following code performs an update.

I think that the CurrentRevision must be set to `0`. In that case we could simply update the manifest [at](https://github.com/openshift/origin/blob/12142d1cf918d8a5678639ba5555af8b503ebeab/test/extended/testdata/apiserver/operator-kube-apiserver-cr.yaml#L8) - wdyt ?

Reply:
Actually with this PR https://github.com/openshift/api/pull/2208 (and https://github.com/openshift/api/pull/2203), currentRevision must not exist (so currentRevision=0 is invalid as well during creation).